packages/editor/src/components/rich-text/index.native.js

```diff
@@ -21,12 +21,20 @@ import { children } from '@wordpress/blocks';
 import FormatToolbar from './format-toolbar';
 import { FORMATTING_CONTROLS } from './formatting-controls';
 
+export function getFormatValue( formatName ) {
```

Comment:
@gziolo I wonder why we need to export this function here. Can 't it be moved in `RichText` component since it's only used there?

Reply:
I think it is exported only to be used in tests